app/javascript/mastodon/actions/pin_statuses.js

```diff
@@ -0,0 +1,38 @@
+import api from '../api';
+
+export const PINNED_STATUSES_FETCH_REQUEST = 'PINNED_STATUSES_FETCH_REQUEST';
+export const PINNED_STATUSES_FETCH_SUCCESS = 'PINNED_STATUSES_FETCH_SUCCESS';
+export const PINNED_STATUSES_FETCH_FAIL = 'PINNED_STATUSES_FETCH_FAIL';
+
+export function fetchPinnedStatuses() {
+  return (dispatch, getState) => {
+    dispatch(fetchPinnedStatusesRequest());
+
+    api(getState).get('/api/v1/accounts/1/statuses', { params: { pinned: true } }).then(response => {
```

Comment:
`'/api/v1/accounts/1/statuses'`

I think that it is necessary to access with the logged in user.

Reply:
Thanks! I totally forgot that